dotcom-rendering/src/components/Subheading.tsx

```diff
@@ -0,0 +1,113 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	ArticleDesign,
+	ArticleDisplay,
+	ArticleSpecial,
+	Pillar,
+} from '@guardian/libs';
+import type { FontWeight } from '@guardian/source-foundations';
+import { from, headline, space } from '@guardian/source-foundations';
+import { textSans } from '@guardian/source-foundations/cjs/source-foundations/src/typography/api';
+import type { ReactNode } from 'react';
+import { palette } from '../palette';
+
+interface Props {
+	id?: string;
+	format: ArticleFormat;
+	topPadding: boolean;
+	children: ReactNode;
+}
```

Comment:
Should we keep this closer to where it’s being used, i.e. the `Subheading` component.

Reply:
done